driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/bpf.bpf.c

```diff
@@ -15,12 +15,12 @@ int BPF_PROG(bpf_e,
 	     long id)
 {
 	struct ringbuf_struct ringbuf;
-	if(!ringbuf__reserve_space(&ringbuf, ctx, BPF_E_SIZE))
+	if(!ringbuf__reserve_space(&ringbuf, ctx, BPF_E_SIZE, PPME_SYSCALL_BPF_2_E))
```

Comment:
that's a good point, there are some real corner cases in which they are not used together, see the `hotplug.bpf.c` file, btw yes we can think of merging them one day, this would simplify the flow, but i would postpone it in another PR :)

Reply:
Yep, it will be part of the aforementioned big `event_table` events sizes refactor i think, if we'll ever get to it :D